gadget/ondisk.go

```diff
@@ -101,165 +55,103 @@ type OnDiskVolume struct {
 	Schema string
 	// size in bytes
 	Size quantity.Size
+	// UsableSectorsEnd is the end (exclusive) of usable sectors on the disk,
+	// this sector specifically is not usable for partitions, though it may be
+	// used for i.e. GPT header backups on some disks. This should be used when
+	// calculating the size of an auto-expanded partition instead of the Size
+	// parameter which does not take this into account.
+	UsableSectorsEnd uint64
```

Comment:
```suggestion
	UsableSectorsEnd quantity.Offset
```

Reply:
As mentioned above, it's misleading since Offset says it is in units of bytes which is not how it is being used here